src/main/steps/screen-questions/language-preference/content.ts

```diff
@@ -13,18 +20,25 @@ const cy: typeof en = {
   line2: 'Dewisiwch pa iaith yr hoffech eu cael.',
   yes: 'Cymraeg a Saesneg',
   no: 'Saesneg yn unig',
+  errors: {
+    languagePreferenceWelsh: {
+      required: 'Dewisiwch yr iaith yr hoffech gael eich dogfennau a’ch gohebiaeth.'
+    }
+  }
 };
 
 export const languagePreferenceForm = {
-  fields: {
-    languagePreferenceWelsh: {
+  fields: [
+    { name: 'languagePreferenceWelsh',
       type: 'radios',
       values: [
         { label: l => l.no, value: 'No' },
         { label: l => l.yes, value: 'Yes' }
-      ]
+      ],
+      required: true,
```

Comment:
as mentioned above, I think this probably isn't necessary as you can just use the presence of the validator to say whether it's required or not